src/date-picker/src/DatePicker.tsx

```diff
@@ -506,7 +511,12 @@ export default defineComponent({
     function openCalendar (): void {
       if (mergedDisabledRef.value || mergedShowRef.value) return
       doUpdateShow(true)
-      if (props.type === 'month' || props.type === 'year') {
+      const { type } = props
+      if (
+        type === 'month' ||
+        type === 'year' ||
+        type === 'quarter'
+      ) {
         void nextTick(scrollYearMonth)
```

Comment:
```suggestion
        void nextTick(scrollYearMonth)
```

按照含义来说得换个名字了吧？scrollPickerColumns 或者啥的